Deleting workspaces with resources under management (#146)
hashicorp/terraform-docs-common: website/docs/cloud-docs/api-docs/changelog.mdx

Keep track of changes to the API for Terraform Cloud and Terraform Enterprise.

## 2022-10-17

* Updated the [Organizations API](/cloud-docs/api-docs/organizations) with the `allow-force-delete-workspaces`, which controls whether workspace administrators can delete workspaces with resources under management.

> * Updated the [Workspacess API](/cloud-docs/api-docs/workspaces) with the new "safe delete" API, which guards against deleting workspaces with resources under management.

Reviewer: ```suggestion
* Updated the [Workspacess API](/cloud-docs/api-docs/workspaces) with a safe delete endpoint that guards against deleting workspaces that are managing resources.
```
